security-domain-to-domain/pom.xml

```diff
@@ -149,10 +153,68 @@
             <plugin>
                 <groupId>org.wildfly.plugins</groupId>
                 <artifactId>wildfly-maven-plugin</artifactId>
+                <version>${version.plugin.wildfly}</version>
                 <configuration>
                     <skip>true</skip>
                 </configuration>
             </plugin>
         </plugins>
     </build>
+    <profiles>
+        <profile>
+            <id>provisioned-server</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.wildfly.plugins</groupId>
+                        <artifactId>wildfly-maven-plugin</artifactId>
+                        <configuration>
+                            <feature-packs>
+                                <feature-pack>                                    
+                                    <location>org.wildfly:wildfly-galleon-pack:${version.server}</location>
+                                </feature-pack>
+                            </feature-packs>
+                            <layers>
+                                <!-- layers may be used to customize the server to provision -->
+                                <layer>cloud-server</layer>
```

Comment:
Isn't the ejb layer needed too?

Reply:
+1 it's strange to me that the tests don't fail.

Reply:
The profiles are in wrong pom.xml, this a multi module maven project.

Reply:
As a matter of fact this one is different than everything else so far, the deployment dir is the "ear" module. I wonder if we need to drop that (and instead pack the ejb in the war) to use server provisioning, will try to find time to investigate this later today.

Reply:
@PrarthonaPaul due to ^^ I may end up sending you a PR to fix everything, will let you know after investigation

Reply:
Sounds good! 
Thanks, everyone for noticing it!